core/scripts/cli/sponsor/create.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require("inquirer");
+const getDefaultAccount = require("../wallet/getDefaultAccount");
+
+const create = async (web3, artifacts, emp) => {
+  const ExpandedERC20 = artifacts.require("ExpandedERC20");
+  const { toWei, fromWei, toBN } = web3.utils;
+
+  const sponsorAddress = await getDefaultAccount(web3);
+
+  const scalingFactor = toBN(toWei("1"));
+  // TODO: Understand why we need a .rawValue in one case but not the other.
+  const totalPositionCollateral = toBN((await emp.totalPositionCollateral()).rawValue.toString());
```

Comment:
This is strange, I would expect you would need `.rawValue` in both instances

Reply:
Yeah no idea why. Took a bunch of trial and error to even get this to work.

Reply:
@chrismaree @mrice32 any idea?